src/Test/Utilities/Portable/Diagnostics/DiagnosticExtensions.cs

```diff
@@ -369,5 +369,11 @@ public static string Stringize(this Diagnostic e)
             retVal = e.Severity.ToString() + " " + e.Id + ": " + e.GetMessage(CultureInfo.CurrentCulture);
             return retVal;
         }
+
+        public static string ToString(this Diagnostic d, IFormatProvider formatProvider)
+        {
+            IFormattable formattable = d;
+            return formattable.ToString(null, formatProvider);
```

Comment:
`format: null`?

Reply:
Sorry. I pushed a bunch of commits to the wrong branch. This wasn't meant to be for this PR.